stable/grafana/templates/deployment.yaml

```diff
@@ -80,21 +95,6 @@ spec:
           volumeMounts:
             - name: sc-dashboard-volume
               mountPath: {{ .Values.sidecar.dashboards.folder | quote }}
-{{- end}}
-{{- if .Values.sidecar.datasources.enabled }}
-        - name: {{ template "grafana.name" . }}-sc-datasources
-          image: "{{ .Values.sidecar.image }}"
-          imagePullPolicy: {{ .Values.sidecar.imagePullPolicy }}
-          env:
-            - name: LABEL
-              value: "{{ .Values.sidecar.datasources.label }}"
-            - name: FOLDER
-              value: "/etc/grafana/provisioning/datasources"
-          resources:
-{{ toYaml .Values.sidecar.resources | indent 12 }}
```

Comment:
Can this resources configuration please be added to the init container too?

Reply:
The resource configuration has been added to the pull request.  A bit confused by the automatic "pull request successfully merged" without seeing it end up in the master branch, but glad to get it moving forward.